src/common/options/mds.yaml.in

```diff
@@ -65,15 +65,6 @@ options:
   - mds
   flags:
   - runtime
-# max xattr kv pairs size for each dir/file
-- name: mds_max_xattr_pairs_size
-  type: size
-  level: advanced
-  desc: maximum aggregate size of extended attributes on a file
-  default: 64_K
-  services:
-  - mds
-  with_legacy: true
```

Comment:
Maybe we should update the `PendingReleaseNotes` about this.

Reply:
Ah, right.  And I didn't even grep'ed for any other documentation that may refer to this setting (e.g. man pages).  I'll need to do that too, I was just waiting until the moment to remove the 'Draft' status of this PR.  I can do that now, in the next update.  I'll give it some more time in case someone has any further comments.

Reply:
IMO, we are abandoning the options from mds and switched to use the mdsmap one, so we should make it clear what has happened in `PendingReleaseNotes`.